packages/office-ui-fabric-react/src/components/DetailsList/DetailsRow.scss

```diff
@@ -12,6 +12,7 @@ $unselectedHoverColor: $ms-color-neutralLighter;
   display: inline-block;
   min-width: 100%;
```

Comment:
Not sure why min-width is set, that was there before. It looks like min-width is being overwritten in both GroupedList and DetailsList programmatically.  Should I remove it?